src/TraceEvent/EventPipe/EventPipeTraceEventParser.cs

```diff
@@ -1,3 +1,4 @@
+#define SUPPORT_V1_V2
```

Comment:
We now need this define in two files, we should move it to the TraceEvent.csproj file under DefineConstants.